src/net/sourceforge/kolmafia/utilities/PHPMTRandom.java

```diff
@@ -59,15 +60,17 @@ public int next(int bits) {
     return (int) (temper(value) >> 1);
   }
 
+  @Override
   public double nextDouble() {
     return nextInt() / (Integer.MAX_VALUE + 1.0);
   }
 
+  @Override
   public int nextInt(final int max) {
-    return nextInt(0, max);
+    return nextBoundedInt(0, max);
```

Comment:
Would you mind making the non-override changes in a separate PR?

(This file, PHPRandom.java, and VotingBoothManager.java)

Reply:
This is sort of part of the override changes - pmd says that `int nextInt(int int)` is an override and needs annotation and fails the build. It's not an override, so if I add that annotation my IDE highlights it as an issue. I think it's a pmd issue but this seemed like the easiest way to solve it.

Reply:
Are you sure about that? I ran pmd locally and for PHPRandom, I see `int nextInt(int)` is an override (correct), but nothing about `int nextInt(int, int)`.

Reply:
I am sure at least that it raised a false positive on my end. I'll revert the change and if it doesn't happen on the CI I'll commit it and look into why it's doing so for me only